docs/conf.py

```diff
@@ -59,19 +53,30 @@
     'sphinx_design',
     'reno.sphinxext',
     'sphinx_copybutton',
+    'hoverxref.extension',
 ]
 
 autosummary_generate = True
+numfig = True
+
+hoverxref_roles = ['term']
+hoverxref_role_types = {'term': 'tooltip'}
```

Comment:
you'll want to enable mathjax as well:
```suggestion
hoverxref_role_types = {'term': 'tooltip'}
hoverxref_mathjax = True
```

Reply:
yes, just did as part of the [followup PR](https://github.com/dwavesystems/dwave-ocean-sdk/pull/324/commits/0d4715faaec2ff245069872300f5645f26abbb40), thanks for pointing it out!